service/matching/matchingEngine.go

```diff
@@ -215,15 +228,19 @@ func (e *matchingEngineImpl) AddDecisionTask(
 	domainID := request.GetDomainUUID()
 	taskListName := request.TaskList.GetName()
 	taskListKind := request.TaskList.Kind
+	taskListType := persistence.TaskListTypeDecision
 
-	e.logger.Debug(
-		fmt.Sprintf("Received AddDecisionTask for taskList=%v, WorkflowID=%v, RunID=%v, ScheduleToStartTimeout=%v",
-			request.TaskList.GetName(),
-			request.Execution.GetWorkflowID(),
-			request.Execution.GetRunID(),
-			request.GetScheduleToStartTimeoutSeconds()))
```

Comment:
`GetScheduleToStartTimeoutSeconds` was dropped, is this intended?

Reply:
yes, this is intentional